app/src/pages/playground/Playground.tsx

```diff
@@ -150,17 +151,17 @@ function PlaygroundContent() {
             >
               <View height="100%" padding="size-200">
                 <Flex direction="row" gap="size-200">
-                  {instances.map((instance, i) => (
+                  {instances.map((instance) => (
                     <div
-                      key={i}
+                      key={instance.id}
                       css={css`
                         flex: 1 1 0px;
```

Comment:
Can we add some flex gap?

<img width="746" alt="image" src="https://github.com/user-attachments/assets/84f6d764-6246-47ca-bc4d-ae704573a8df">

Reply:
ahh yes thank you i saw this then forgot to circle back